nilearn/surface/surface.py

```diff
@@ -2048,3 +2077,76 @@ def extract_data(img, index):
         .reshape(mesh.parts[hemi].n_vertices, last_dim)
         for hemi in data.parts
     }
+
+
+def compute_adjacency_matrix(mesh, values="ones", dtype=None):
```

Comment:
technically covered by tests but actually unused in practice

Reply:
to be removed then...